jupyterlab_comments/jupyterlab_comments/git_commands.py

```diff
@@ -41,6 +41,20 @@ def status(self, repo_path):
 	def appraise_pull(self, git_root_dir):
 		self.run(git_root_dir, 'appraise', 'pull', self.remote)
 
+	def appraise_push(self, git_root_dir):
+		self.run(git_root_dir, 'appraise', 'push', self.remote)
+
+	def push_local_comments(self, git_root_dir):
+		self.run(git_root_dir, 'appraise', 'pull', self.remote)
+		return_code = subprocess.call(['git', 'appraise', 'push', self.remote], cwd=git_root_dir)
+		#Retry once if refs fail to get pushed
+		if return_code == 1:
```

Comment:
do we always get return code = 1? or we could get another non-zero value?

Reply:
thanks for pointing this out! I'm changing this so it catches all non-zero return code values